src/main/java/org/fcrepo/transform/transformations/LDPathTransform.java

```diff
@@ -157,15 +157,4 @@ public boolean equals(final Object other) {
     public int hashCode() {
         return Objects.hashCode(query);
```

Comment:
Just as a side note, I suspect this is pretty wrong. It doesn't take into account much of the state involved.